### PR TITLE
[#398] Fix filter is not working in manage users page

### PR DESCRIPTION
**Pull request by ifirmawan**
## TODO/DONE

- [x] Fix filter after Open user details/change
- [x] Fix filter is gone
- [x] Fix Reset button is gone

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205219328247491

### CODE REVIEW

File: frontend/src/pages/admin/ManageUser.jsx [hunk 1]
```diff
@@ -95,16 +96,25 @@ const ManageUser = () => {
       });
   };
 
-  const onSearch = (values) => {
+  const onSearch = ({ search, organisation, role }) => {
+    const values = Object.fromEntries(
+      Object.entries({ search, organisation, role }).filter(
+        ([key, value]) => value // eslint-disable-line no-unused-vars
```
Comment (dedenbangkit) [inline frontend/src/pages/admin/ManageUser.jsx:102]: use _ @ifirmawan

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: frontend/src/pages/admin/ManageUser.jsx (hunk 1 above)
Comment (ifirmawan) [reply frontend/src/pages/admin/ManageUser.jsx:102]: Okay mas, on it

Comment (dedenbangkit) [reply frontend/src/pages/admin/ManageUser.jsx]: no worries, I just pushed the changes

**Review by dedenbangkit — CHANGES_REQUESTED** [review]
# New error case 1
- Select an Organisation as filter -> Search
- Enable / Disable Pending User

# New error case 2
- Search person
- Enable / Disable Pending User

**Comment by ifirmawan** [comment]
> # New error case 1
> * Select an Organisation as filter -> Search
> * Enable / Disable Pending User
> 
> # New error case 2
> * Search person
> * Enable / Disable Pending User

Hi mas @dedenbangkit, I've fixed the new errors above. Thank you

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**